docs/essentials/highlight.md

```diff
@@ -32,6 +32,12 @@ To highlight DOM elements with the addon, you'll need to emit the `HIGHLIGHT` ev
 
 </div>
 
+<div class="aside">
+
+ℹ️  The `emit` function derived from the `useChannel` API hook creates a communication channel in Storybook's UI to listen for events and update the UI accordingly. The Highlight addon uses this channel to listen to custom events and update the highlighted elements (if any) accordingly.
+
+</div>
+
```

Comment:
@ATREAY sorry for not being clear on this. In my previous comment, when I mentioned moving the aside above to the "Reset highlighted items" section, I was referring to applying the following change to the documentation:

```md
### Reset highlighted elements

Out of the box, Storybook automatically removes highlighted elements when transitioning between stories. However, if you need to clear them manually, you can emit the `RESET_HIGHLIGHT` event from within a story or an addon. For example:

<!-- prettier-ignore-start -->

<CodeSnippets
  paths={[
    'react/addon-highlight-reset.js.mdx',
    'react/addon-highlight-reset.ts.mdx',
    'angular/addon-highlight-reset.ts.mdx',
    'vue/addon-highlight-reset.js.mdx',
    'vue/addon-highlight-reset.ts.mdx',
    'web-components/addon-highlight-reset.js.mdx',
    'web-components/addon-highlight-reset.ts.mdx',
  ]}
/>

<!-- prettier-ignore-end -->

<div class="aside">

ℹ️  The `emit` function derived from the `useChannel` API hook creates a communication channel in Storybook's UI to listen for events and update the UI accordingly. The Highlight addon uses this channel to listen to custom events and update the highlighted elements (if any) accordingly.

</div>
```

Reply:
@ATREAY almost got it, just needs to be placed in the right spot and this is good to go.